nautobot/wireless/choices.py

```diff
@@ -0,0 +1,345 @@
+from nautobot.core.choices import ChoiceSet
+
+
+class ChannelWidthChoices(ChoiceSet):
```

Comment:
Should these choicesets have the model that they're used for in the name? (`RadioProfileChannelWidthChoices`, `RadioProfileFrequencyChoices`)

Reply:
That's fine, I didn't realize there was a standard on that. I will change.